### PR TITLE
add new fields to edit project form

### LINKED ISSUE #1489: Add in all available fields from the vrms database on the project edit page
### Overview
bonnie requested to have all available fields in the db editable for admin level users

### Action Items
- [ ] add text inputs on the project page corresponding with the project model

### Resources/Instructions
example part of a json object from get request of vrms db
``` diff
"recruitingCategories": [],
+ "partners": [ ""
],
+ "managedByUsers": [],
"_id": "5eec35...",
"name": "Community of Practice: Data Science",
"description": "",
"githubIdentifier": "",
+ "projectStatus": "Active",
+ "location": "Remote",
"githubUrl": "https://github.com/hackforla/website",
"slackUrl": "",
+ "googleDriveUrl": "",
+ "googleDriveId": "",
"hflaWebsiteUrl": "https://www.hackforla.org/projects/website",

+ "createdDate": "2020-06-19T03:47:56.782Z", (display)
```

### PR DESCRIPTION
**Pull request by bkmorgan3**
Fixes #1489 

### What changes did you make and why did you make them ?

  - created new array in data.js representing the fields not displayed in the Model
 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

Screenshot 2023-10-11 at 1 16 18 PM

### CODE REVIEW

File: client/src/components/manageProjects/editProject.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ const EditProject = ({
         />
       </div>
       <ProjectForm
-        arr={simpleInputs}
+        arr={[...simpleInputs, ...additionalInputsForEdit]}
```
Comment (trillium) [inline client/src/components/manageProjects/editProject.js:100]: In order for state to be passed to the form the info needs to be destructed off `projectToEdit`

```js
const [formData, setFormData] = useState({
    name: projectToEdit.name,
    description: projectToEdit.description,
    location: projectToEdit.location,
    githubIdentifier: projectToEdit.githubIdentifier,
    githubUrl: projectToEdit.githubUrl,
    slackUrl: projectToEdit.slackUrl,
    googleDriveUrl: projectToEdit.googleDriveUrl,
    hflaWebsiteUrl: projectToEdit.hflaWebsiteUrl,
    // ------ Need these fields to be destructured
    partners: projectToEdit.partners,
    managedByUsers: projectToEdit.managedByUsers,
    projectStatus: projectToEdit.projectStatus,
    googleDriveId: projectToEdit.googleDriveId,
    createdDate: projectToEdit.createdDate,
    // ------ 
  });

File: client/src/components/data.js [hunk 2]
```diff
@@ -54,4 +54,37 @@ export const simpleInputs = [
...
+  },
+   {
+    label: 'Google Drive ID',
+    name: 'googleDriveId',
+    type: 'text',
+    placeholder: 'htttps://drive.google.com/',
+  },
+  {
+    label: 'Created at',
+    name: 'createdAt',
```
Comment (trillium) [inline client/src/components/data.js]: typo, correct field name is `createdDate`

File: client/src/components/data.js [hunk 3]
```diff
@@ -54,4 +54,37 @@ export const simpleInputs = [
...
+export const additionalInputsForEdit = [
+  {
+    label: 'Partners',
+    name: 'partners',
+    type: 'text',
+    placeholder: 'partners',
+  },
+  {
+    label: 'Managed by Users',
+    name: 'mangedByUsers',
```
Comment (trillium) [inline client/src/components/data.js]: typo, correct field name is `managedByUsers`

**Review by trillium — CHANGES_REQUESTED** [review]
Requested some changes on typos, information destructuring.

I think we also need to discuss if we want to let the user edit a field like `createdDate`

**Comment by bkmorgan3** [comment]
> Requested some changes on typos, information destructuring.
> 
> I think we also need to discuss if we want to let the user edit a field like `createdDate`

@Spiteless  All things fixed.  I disabled the createdDate field.

**Review by trillium — APPROVED** [review]
Thanks for the updates, approved :)

**Pull request merged**